apps/frontend/core-dapp/src/features/trade/TradeStore.ts

```diff
@@ -871,6 +871,36 @@ export class TradeStore {
     return this.userClosePositionValue.value
   }
 
+  get ppoRewardForOpenTrade(): string | undefined {
+    const { action, openTradeAmountBN, paymentToken } = this
+    const { depositFee, percentDenominator } = this.root.collateralStore
+
+    if (action !== 'open' || paymentToken.type !== 'native') return '0'
```

Comment:
Don't really need this right because ppo reward isn't shown in close position summary?
```suggestion
    if (paymentToken.type !== 'native') return '0'
```

Reply:
Yea, this was just being defensive. I can remove it, but I would find it helpful as a new developer onboarded to the codebase. Sometimes the `TradeStore` can be confusing, and you don't know if a value applies to open, close, or both. In fact, I'd argue that they should be separate stores that reuse logic.

Reply:
okiee sounds good, at some point we should consider splitting it up to CloseTradeStore, OpenTradeStore, and TradeStore should only store logics like selectedMarket, selectedPosition